src/NServiceBus.Core/Pipeline/ConnectorContextExtensions.cs

```diff
@@ -159,6 +215,10 @@ public static IAuditContext CreateAuditContext(this ForkConnector<IIncomingPhysi
         /// </summary>
         public static IForwardingContext CreateForwardingContext(this ForkConnector<IIncomingPhysicalMessageContext, IForwardingContext> forwardingContext, OutgoingMessage message, string forwardingAddress, IIncomingPhysicalMessageContext sourceContext)
         {
+            Guard.AgainstNull(nameof(message), message);
+            Guard.AgainstNull(nameof(forwardingAddress), forwardingAddress);
```

Comment:
I don't know :-)

The test is very coarse, it only tests to see if some "guard" is in the instruction list of the method.

What is the logical rule here? Can the forwarding address by empty? Can it be white space? I'd expect at least `ForwardingContext` to be capturing these invariants.